docs/troubleshooting-guide/cant-see-tables.md

```diff
@@ -0,0 +1,85 @@
+# I can't see my tables
```

Comment:
Should we include a table of contents to give people an overview of the potential causes?

Reply:
Good point